spec/features/reviews/administrator_can_delete_review_spec.rb

```diff
@@ -0,0 +1,52 @@
+require "rails_helper"
+
+feature 'administrator can delete review', %{
+  As an administrator
+  I want to delete a review
+  Because it is inappropriate and/or offensive
+
+  Acceptance Criteria
+  [x] Admin can delete a review
+  [x] Review is no longer on bar details page
+  [x] Admin receives confirmation review has been deleted
+  [x] Nonadmin cannot delete a review
+  } do
+
+  scenario 'admin can delete a review' do
+    bar = FactoryGirl.create(:bar)
+    admin = FactoryGirl.create(:user, admin: true)
+    FactoryGirl.create(:review, bar: bar)
+
+    sign_in_as(admin)
+    visit root_path
+    click_link (bar.name)
+
+    expect(page).to have_content('Delete Review')
+  end
+
+  scenario 'admin can delete a review' do
```

Comment:
You need to make sure to have scenarios are different. `Admin can delete a review` should be a single test and the next one should be another test for something else.